core/src/navigation/ContextSwitcher.html

```diff
@@ -0,0 +1,194 @@
+    helpers: {
+      getMatchingNodeName: (options, config, id) => {
+        return ContextSwitcherHelpers.getMatchingNodeName(options, config, id);
+      }
+    },
+    methods: {
+      async fetchOptions() {
+        let options = this.get().options;
+        if (!options.length || getConfigValue('navigation.contextSwitcher.lazyloadOptions') !== false) {
+          const contextSwitcherOptions = await getConfigValueAsync('navigation.contextSwitcher.options');
+          options = await ContextSwitcherHelpers.generateSwitcherNav(this.get().config, contextSwitcherOptions);
+          this.set({options});
+        }
+      },
+      async setSelectedContext(currentPath) {
+        let selectedLabel;
+        const parentNodePath = this.get().config.parentNodePath;
+        const inContextBasePath = parentNodePath && currentPath.indexOf(parentNodePath) !== -1;
+        if (inContextBasePath) {
+          // we are inside the context switcher base path
+          const truncatedPath = trimLeadingSlash(currentPath.replace(parentNodePath, ''));
+          const selectedId = truncatedPath.split('/')[0];
+          selectedLabel = await ContextSwitcherHelpers.getMatchingNodeName(this.get().options, this.get().config.fallbackLabelResolver, selectedId);
+        }
+        this.set({
+          selectedLabel
+        })
+      },
+      async onActionClick(node) {
+        if (node.clickHandler) {
+          const result = await getConfigValueFromObjectAsync({result: node.clickHandler(node)}, 'result');
+          if (!result) {
+            // If the clickHandler returns falsy value, do nothing, else continue with linking
+            return;
+          }
+        }
+        this.goTo(node.link);
+      },
+      goTo(path) {
+        if (getConfigBooleanValue('routing.useHashRouting')) {
+          window.location.hash = path;
+        } else {
+          window.location.pathname = path;
+        }
+      },
+      onContextSwitcherOpen(name) {
+        this.toggleDropdownState(name);
+        this.fetchOptions();
+      },
+      toggleDropdownState(name) {
+        const dropDownStates = this.get().dropDownStates || {};
+        dropDownStates[name] = getNegatedBoolString(dropDownStates[name]);
+        this.set({
+          dropDownStates
+        });
+      },
+      closeDropdown(name) {
+        const dropDownStates = this.get().dropDownStates || {};
+        dropDownStates[name] = 'false';
+        this.set({
+          dropDownStates
+        });
+      }
+    }
+  };
+</script>
+
+<style type="text/scss">
+  $fd-icons-path: 'node_modules/fundamental-ui/scss/icons/';
+  @import 'node_modules/fundamental-ui/scss/icons';
+  // @import 'node_modules/fundamental-ui/scss/components/action-bar';
+  // @import 'node_modules/fundamental-ui/scss/components/global-nav';
+  // @import 'node_modules/fundamental-ui/scss/components/popover';
+  // @import 'node_modules/fundamental-ui/scss/components/button';
+  // fundamental-ui is loaded globally
```

Comment:
Probably I am missing something, but here we are in Luigi Core and I see fundamental-ui is loaded globally in the angular example. We have to load it inside Luigi Core itself, not in the app that uses Luigi.